composer.json

```diff
@@ -45,10 +45,10 @@
       "phar:build": "env PATH=tools:$PATH box build",
       "update-class-lists": ["\\Terminus\\UpdateClassLists::update"],
       "behat": "SHELL_INTERACTIVE=true behat --colors --config tests/config/behat.yml --suite=default",
-      "cbf": "phpcbf --standard=PSR2 -n tests/unit_tests/* bin/terminus src/*",
```

Comment:
The `-n` option ignores warnings. The standards file uses PSR-2, then removes the line-length rule. Files defining custom rules are then located based on the location of the standards file. The custom rule it will locate extends the line-length rule but ignores lines inside comment brackets:
```
/**
 * Like these
 */
```
It is necessary to ignore the line-length rule in the comments because the comments are used to generate the Terminus commands and their help documentation.